autoload/direnv.vim

```diff
@@ -8,28 +8,25 @@ let s:direnv_cmd = get(g:, 'direnv_cmd', 'direnv')
 let s:direnv_interval = get(g:, 'direnv_interval', 500)
 let s:direnv_max_wait = get(g:, 'direnv_max_wait', 5)
 let s:direnv_auto = get(g:, 'direnv_auto', 1)
-let s:job_status = { 'running': 0, 'stdout': [], 'stderr': [] }
+let s:job_status = { 'running': 0, 'stdout': [''], 'stderr': [''] }
 
 function! direnv#auto() abort
   return s:direnv_auto
 endfunction
 
 function! direnv#on_stdout(_, data, ...) abort
-  call extend(s:job_status.stdout, a:data)
+  let s:job_status.stdout[-1] .= a:data[0]
+  call extend(s:job_status.stdout, a:data[1:])
 endfunction
 
 function! direnv#on_stderr(_, data, ...) abort
-  call extend(s:job_status.stderr, a:data)
+  let s:job_status.stderr[-1] .= a:data[0]
+  call extend(s:job_status.stderr, a:data[1:])
 endfunction
 
 function! direnv#on_exit(_, status, ...) abort
   let s:job_status.running = 0
 
-  for l:m in s:job_status.stderr
```

Comment:
@hauleth  Why did you delete these lines? This makes errors invisible if needed.

mater branch now has `g:direnv_silent_load` for this purpose. Can you rebase from it?